tests/std/tests/Dev10_851347_weak_ptr_virtual_inheritance/test.cpp

```diff
@@ -66,6 +66,7 @@ void test_owner_equal() {
 }
 
 void test_gh_000258_sometimes_avoid_locking() {
+    // weak_ptr's converting constructors could sometimes avoid locking

```

Comment:
I think it is unnecessary repetition of function name and comment, and since the comment is clearer, suggest changing function name.